upgrades/CHANGELOG.md

```
@@ -40,6 +40,14 @@ Goal of the upgrade here
 Add new upgrades after this line based on the template above
 -----
 
+##### v2.1.0
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h5

(MD001, heading-increment)